dags/aggregates_python.py

```diff
@@ -64,17 +45,12 @@ def main_of_validation(ti):
 
 def main_of_aggregates_creation(ti):
     """Main function for aggregates creation."""
-    # Set connection parameters
-
-    postgres_user = os.environ.get('POSTGRES_USER')
-    postgres_password = os.environ.get('POSTGRES_PASSWORD')
-    postgres_db = os.environ.get('POSTGRES_DB')
 
-    superior_aggregates_creator(postgres_user, postgres_password, postgres_db)    
-
-# Adding the attachment (make sure the file exists before sending the email)
-attachments = [absolute_path]
+    superior_aggregates_creator()    
 
+def main_of_send_email_with_attachment(ti):
```

Comment:
i dont understand why we are creating this "main" functions than insted call then directly in Tasks

Reply:
Ok, I will use email_callback instead